shared/sys/sys_main.cpp

```diff
@@ -199,10 +199,8 @@ static void Sys_ErrorDialog( const char *error )
 		fclose( fp );
 
 		const char *errorMessage = va( "%s\n\nThe crash log was written to %s", error, crashLogPath );
-		if ( SDL_ShowSimpleMessageBox( SDL_MESSAGEBOX_ERROR, "Error", errorMessage, NULL ) < 0 )
-		{
-			fprintf( stderr, "%s", errorMessage );
```

Comment:
My use case is to easy copy/paste the filen ame and open it to find the error. That's not possible with the message box.